editors/nvi/files/patch-common_conv.h.diff

```diff
@@ -0,0 +1,9 @@
+--- common/conv.h.orig	2023-09-22 20:14:30.000000000 -0700
++++ common/conv.h	2023-09-22 20:15:09.000000000 -0700
+@@ -21,3 +21,6 @@
+ 	char2wchar_t	input2int;
+ 	wchar2char_t	int2disp;
+ };
++
++void conv_init (SCR *orig, SCR *sp);
++int conv_enc (SCR *sp, int option, char *enc);
```

Comment:
Looks like this is fixing implicit declaration of function errors? If so, write that at the top of the patchfile. In other ports, patchfiles that fix implicit declaration of function errors are often named implicit.patch.

Reply:
I tried to rename this patch implicit.patch but for some reason when I did this it tried to download the patch instead of using the one in the files directory.  So, I've left the name of the patch the same, but I added a note to the preamble of the patch to explain the implicit declarations error that are being fixed.